test/integration/targets/junos_netconf/tests/utils/junos_command.yaml

```diff
@@ -0,0 +1,8 @@
+---
+- name: run junos_command to check netconf connectivity
+  junos_command:
+    rpcs: get-software-information
+    provider: "{{ netconf }}"
```

Comment:
`provider` shouldn't be necessary to keep here, nor `port`, I think

Reply:
Fixed